flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/rules/physical/batch/BatchPhysicalOverAggregateRule.scala

```diff
@@ -190,7 +190,8 @@ class BatchPhysicalOverAggregateRule
     val inputNameList = inputType.getFieldNames
     val inputTypeList = inputType.getFieldList.asScala.map(field => field.getType)
 
-    val aggNames = aggCalls.map(_.getName)
+    // we should avoid duplicated names with input column names
+    val aggNames = aggCalls.map(c => RowTypeUtils.getUniqueName(c.getName, inputNameList))
```

Comment:
what if there are three duplicate names: "a, a, a", the result will be "a, a0, a0". 
the new unique name should be added to inputNameList.

please add an UT for this class

Reply:
please add a plan test for this change